doc/source/whatsnew/v1.4.0.rst

```diff
@@ -399,6 +399,7 @@ Other API changes
 ^^^^^^^^^^^^^^^^^
 - :meth:`Index.get_indexer_for` no longer accepts keyword arguments (other than 'target'); in the past these would be silently ignored if the index was not unique (:issue:`42310`)
 - Change in the position of the ``min_rows`` argument in :meth:`DataFrame.to_string` due to change in the docstring (:issue:`44304`)
+- Reduction operations for :class:`DataFrame` or :class:`Series` nor raising a ``ValueError`` when ``None`` is passed for ``skipna`` (:issue:`44178`)
```

Comment:
is this an actual change though? this is just restoring the 1.3.x behavior right?

Reply:
In 1.3.x skipna=None was the default, but it was documented as ``bool, default True``. The None was translated into True internally. This was changed to skipna=True in #43980, but now None was evaluated to False internally. This ensures that we raise for non boolean inputs.